src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -17473,9 +17473,9 @@ partial struct CustomHandler
 
             var comp = CreateCompilation(new[] { code, InterpolatedStringHandlerArgumentAttribute, GetInterpolatedStringCustomHandlerType("CustomHandler", "partial struct", useBoolReturns: false) });
             comp.VerifyDiagnostics(
-                // (5,1): error CS1615: Argument 3 may not be passed with the 'ref' keyword
+                // (5,1): warning CS9501: Argument 3 should not be passed with the 'ref' keyword
```

Comment:
>// (5,1): warning CS9501: Argument 3 should not be passed with the 'ref' keyword [](http://example.com/codeflow?start=16&length=80)

I think we should verify expected behavior for scenarios that became success cases.